lib/oneview-sdk/image-streamer/resource/api300/os_volume.rb

```diff
@@ -0,0 +1,49 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative 'resource'
+
+module OneviewSDK
+  module ImageStreamer
+    module API300
+      # OS Volume resource implementation for Image Streamer
+      class OSVolume < Resource
+        BASE_URI = '/rest/os-volumes'.freeze
+
+        # Method is not available
+        # @raise [OneviewSDK::MethodUnavailable] method is not available
+        def create
```

Comment:
For all these methods that call `unavailable_method`, let's let them take any arguments. See #155.

Otherwise it looks good to me. However, I have no I3S appliance to test against, so I'll let someone else give the final 👍